pkgs/development/python-modules/datasets/default.nix

```diff
@@ -3,6 +3,8 @@
 , fetchFromGitHub
 , dill
 , filelock
+, fsspec
+, huggingface_hub
```

Comment:
```suggestion
, huggingface-hub
```

Reply:
I considered that, but `huggingface_hub` is actually the upstream name.

Reply:
Not sure what the policy is in such cases.

Reply:
We normalize the package names according to some PEP which replaces special characters with like _ with -.

Reply:
Ack, all fixed.